README.md

```diff
@@ -45,6 +45,20 @@ Usually this script is not run after installation but could be used with some mo
 
 This script was originally contributed by [@Landrash](https://github.com/landrash).
 
+### Install MariaDB *(install_mariadb.sh)*
+This script installs MariaDB and it's dependencies for use with the [recorder](https://home-assistant.io/components/recorder/) component in Home Assistant.
+No database or database user is created during this setup and will need to be created manually.
+
+#### Example for creating database  
+```$ mysql -u root -p```  
```

Comment:
Had to put "sudo mysql -u root -p", or else it reported access denied.

Reply:
True. Fixing.